src/commands/flex/plugins/start.ts

```diff
@@ -1,15 +1,66 @@
+import { flags } from '@oclif/command';
+
 import { createDescription } from '../../../utils/general';
-import FlexPlugin from '../../../sub-commands/flex-plugin';
+import FlexPlugin, { ConfigData, SecureStorage } from '../../../sub-commands/flex-plugin';
+
 /**
  * Starts the dev-server for building and iterating on a flex-plugin
  */
 export default class FlexPluginsStart extends FlexPlugin {
-  static description = createDescription('Starts a dev-server to build the Flex plugin locally', true);
+  static description = createDescription('Starts a dev-server to build the Flex plugin locally', false);
+
+  static flags = {
+    ...FlexPlugin.flags,
+    name: flags.string({
+      multiple: true,
+      char: 'n',
+    }),
+    'include-remote': flags.boolean(),
+  };
+
+  constructor(argv: string[], config: ConfigData, secureStorage: SecureStorage) {
+    super(argv, config, secureStorage, {});
+
+    if (this._flags['include-remote'] || this._flags.name) {
+      this.opts.runInDirectory = false;
+    }
+  }
 
   /**
    * @override
    */
   async doRun() {
-    await this.runScript('start');
+    const names = this._flags.name;
+    const includeRemote = this._flags['include-remote'];
+    const flexInput: string[] = [];
+    const pluginInput: string[][] = [];
+
+    for (let i = 0; names && i < names.length; i++) {
+      flexInput.push('--name', names[i]);
+      pluginInput.push(['--name', names[i]]);
+    }
+
+    if (includeRemote) {
+      flexInput.push('--include-remote');
+    }
+
+    if (flexInput) {
+      await this.runScript('start', ['flex', ...flexInput]);
+    }
+
+    for (let i = 0; pluginInput && i < pluginInput.length; i++) {
```

Comment:
`flexInput` and `pluginsInput` must always co-exist (i.e. one cannot exist without the other), so instead do

```ts
if (flexInput.length && pluginsInput.args) {
  await this.runScript('start', ['flex', ...flexInput]);
  // loop and do await this.runScript('start');
} else {
   await this.runScript('start');
}
```

Reply:
Also rename `flexInput` to `flexArgs` and `pluginsInput` to `pluginsArgs`.

Reply:
Also, what is the behavior of `--include-remote` only? Based on this code, if I run `twilio flex:plugins:start --include-remote` you'll only run `await this.runScript('start', ['flex', ...flexInput]);` an nothing else.